### ISSUE TITLE
No value has been specified for property 'javaHome'

### ISSUE THREAD

**Issue by pitcer** [beryx/badass-runtime-plugin#11]
Running `runtime` task on Linux, Java 11, Gradle 5.2.1 and Runtime Plugin 1.1.5 causes the following error:
```
A problem was found with the configuration of task ':runtime'.
> No value has been specified for property 'javaHome'.
```
Using the following configuration in build.gradle:
```
runtime {
	javaHome.set(System.getProperty("java.home"))
}
```
fixes the problem, so I propose to change [this line](https://github.com/beryx/badass-runtime-plugin/blob/c886e87739b09968c1349ac9cd11946f53767a90/src/main/groovy/org/beryx/runtime/data/RuntimePluginExtension.groovy#L58) to `javaHome.set(System.getProperty("java.home"))`. It should work in every case.

**Comment by siordache** [comment]
@pitcer Thanks for reporting this problem. It's fixed in release 1.1.6.
I took the same approach as the [badass-jlink plugin](https://github.com/beryx/badass-jlink-plugin) for setting the default value of the [javaHome property](https://badass-runtime-plugin.beryx.org/releases/latest/#_properties):

- if present, use the value of the `badass.runtime.java.home` system property
- if present, use the value of the `BADASS_RUNTIME_JAVA_HOME` environment variable
- if the `java.home` system property points to a JRE that contains javac, jar, and jlink, use its value
- otherwise, use the value of the `JAVA_HOME` environment variable

**Comment by pitcer** [comment]
Thanks for the fix!
